controllers/idler/idler_controller.go

```diff
@@ -141,6 +155,64 @@ func (r *Reconciler) ensureIdling(logger logr.Logger, idler *toolchainv1alpha1.I
 	return r.updateStatusPods(idler, newStatusPods)
 }
 
+func (r *Reconciler) createNotification(logger logr.Logger, idler *toolchainv1alpha1.Idler) error {
+	//Get the HostClient
+	hostCluster, ok := r.GetHostCluster()
+	if !ok {
+		err := fmt.Errorf("unable to get the host cluster")
+		logger.Error(err, "host Cluster not found")
+		return err
+	}
+	// add sending notification here
+	//check the condition on Idler if notification already sent
+	_, found := condition.FindConditionByType(idler.Status.Conditions, toolchainv1alpha1.IdlerActivatedNotificationCreated)
+	if !found {
+		userEmails := r.getUserEmailFromUserSignup(hostCluster, idler)
+		// Only create a notification if not created before
+		for _, userEmail := range userEmails {
+			_, err := notify.NewNotificationBuilder(hostCluster.Client, hostCluster.OperatorNamespace).
+				WithNotificationType(toolchainv1alpha1.NotificationTypeIdled).
+				WithControllerReference(idler, r.Scheme).
+				WithTemplate("idleractivated").
+				Create(userEmail)
+			if err != nil {
+				return errs.Wrapf(err, "Unable to create Notification CR from Idler")
+			}
+		}
+		// update Condition
+		if err := r.setStatusIdlerNotificationCreated(idler); err != nil {
+			return err
+		}
+	}
+	//notification already created
+	return nil
+}
+
+func (r *Reconciler) getUserEmailFromUserSignup(hostCluster *cluster.CachedToolchainCluster, idler *toolchainv1alpha1.Idler) []string {
```

Comment:
```suggestion
func (r *Reconciler) getUserEmailsFromIdler(hostCluster *cluster.CachedToolchainCluster, idler *toolchainv1alpha1.Idler) []string {
```

Reply:
changed it to getUserEmailFromMUR in [777b019](https://github.com/codeready-toolchain/member-operator/pull/357/commits/777b019b9dc2a2da36de0b0e2754b8012b10f7d8)